client/components/update-post-status/update-template.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+export default function UpdateTemplate( {
+	post,
+	previousStatus,
+	resetToPreviousState,
+	status,
+	strings: s,
+} ) {
+	let updateClass = 'conf-alert';
+	let updateText, undoTemplate, undoClick, trashText;
+
+	switch ( status ) {
+		case 'trashing':
+		case 'deleting':
+			trashText = ( status === 'deleting' ) ? s.deleting : s.trashing;
+			updateText = (
+				<span>{ trashText } <span className="loading-dot">.</span><span className="loading-dot">.</span><span className="loading-dot">.</span></span>
+			);
+			updateClass += ' conf-alert--trashing';
+			break;
+
+		case 'trash':
+			undoClick = resetToPreviousState;
+			undoTemplate = (
+				<a className="undo" onClick={ undoClick }><span>{ s.undo }</span></a>
+			);
+			updateText = s.trashed;
```

Comment:
I guess we were trying to be clever here for page vs post. I suppose we should know the type from the full post object?

So we can maybe update this to
```jsx
// trashed
updateText = translate( 'Moved to Trash' );

// deleteWarning
updateText = translate( 'Delete this %(postType)s permanently?', { args: { postType } } );
```